databuilder/databuilder/extractor/redshift_table_usage_extractor.py

```diff
@@ -0,0 +1,44 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+
+from databuilder.extractor.base_postgres_column_usage_extractor import BasePostgresColUsageExtractor
+
+
+class RedshiftTableUsagextractor(BasePostgresColUsageExtractor):
+    """
+    Extracts Redshift table usage stats from underlying meta store database using SQLAlchemyExtractor
+    This differs from the PostgresMetadataExtractor because in order to support Redshift's late binding views,
+    we need to join the INFORMATION_SCHEMA data against the function PG_GET_LATE_BINDING_VIEW_COLS().
+    """
+
+    def get_sql_statement(self, use_catalog_as_cluster_name: bool, where_clause_suffix: str) -> str:
+        if use_catalog_as_cluster_name:
+            cluster_source = "CURRENT_DATABASE()"
+        else:
+            cluster_source = f"'{self._cluster}'"
+
+        return """
+        select
+            'redshift' as database,
+            {cluster_source} as cluster,
+            trim(sti.schema) as schema,
+            trim(sti.table) as table_name,
+            trim(pu.usename) as user_email,
```

Comment:
Similar comment as above, this is a very opinionated approach to getting the email. For many users there will also need to be some hook to enrich this value with the _real_ email.